Code/RDGeneral/RDGeneralExceptions.cpp

```diff
@@ -0,0 +1,18 @@
+//
+//  Copyright (C) 2021 Schrödinger, LLC
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+/*
+ * This file is just to make sure exceptions in this directory
+ * get built into the library.
+ */
+
+#include "BadFileException.h"
```

Comment:
The problem is that by adding the export macros to the declarations of the exceptions, they become DLL exports/imports in Windows, and as such, they need to be built into the DLL. Which doesn't happen, because these headers aren't imported in any implementation file in RDGeneral (well, I just noticed Exception.h is actually included in Dict.h, so that one is, but not the other two).

But, yeah, I think this might not be the best way to achieve that. I'll add the headers to RDGeneral's CMake build, and remove this file.

Reply:
Ah, no: that doesn't work. CMake just ignores the headers.